docs-internal/Labels.md

```diff
@@ -7,14 +7,15 @@ easier.
 | Name                  | Description                                                                           |
 | --------------------- | ------------------------------------------------------------------------------------- |
 | 💥 breaking change    | A change that will break the API                                                      |
-| 🐛 bug                | An unexpected behavior                                                                |
+| 🐛 bug                | An unexpected behavior of the API                                                     |
 | 📚 docs               | An addition or correction of the API documentation                                    |
-| 🚀 enhancement        | A new feature or request                                                              |
-| 🔮 epic               | A collection of enhancements                                                          |
+| 🚀 new feature        | A change that adds new functionality to the API                                       |
+| 💡 feature update     | An update to existing functionality of the API                                        |
+| 🔮 epic               | A collection of new features                                                          |
 | 🔬 needs confirmation | A bug that needs to be replicated                                                     |
 | 💬 needs discussion   | A feature not yet agreed on that needs more discussion                                |
 | 🙏 good first issue   | An easy issue for a new contributor                                                   |
-| 🚧 tech debt          | A change that does not affect the API, but instead improves the health of the project |
```

Comment:
I think `tech debt` was a bit confusing as it usually refers to code and I would have liked to include under this label changes to internal documentation, workflow updates, etc, as we don't have a label for them.